feature-libs/pickup-in-store/root/pickup-in-store-root.module.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { CmsConfig, provideDefaultConfigFactory } from '@spartacus/core';
+import { STORE_FINDER_FEATURE } from '@spartacus/storefinder/root';
+import {
+  CartPickupOptionsContainerModule,
+  OrderConsignmentContainerModule,
+  PdpPickupOptionsContainerModule,
+} from './components/index';
+
+import {
+  PICKUP_IN_STORE_CORE_FEATURE,
+  PICKUP_IN_STORE_FEATURE,
+} from './feature-name';
+
+export function defaultPickupInStoreComponentsConfig(): CmsConfig {
+  return {
+    featureModules: {
+      [PICKUP_IN_STORE_FEATURE]: {
+        cmsComponents: [
+          'CheckoutReviewPickup',
+          'MyPreferredStoreComponent',
+          'OrderConfirmationPickUpComponent',
+          'PickupInStoreDeliveryModeComponent',
+        ],
+        dependencies: [STORE_FINDER_FEATURE],
+      },
+      [PICKUP_IN_STORE_CORE_FEATURE]: PICKUP_IN_STORE_FEATURE,
+    },
+  };
+}
+
+@NgModule({
+  imports: [
+    CartPickupOptionsContainerModule,
+    PdpPickupOptionsContainerModule,
+    OrderConsignmentContainerModule,
```

Comment:
Why these components are in root?

Reply:
Because they are displayed in outlets, not CMS components, and need to be eagerly loaded. The rest can be lazy loaded later.

Reply:
You can import `OutletModule.forChild()`.

Reply:
We weren't aware of this, and we've asked numerous people if this was possible. After this PR has been merged, we'll look into it but what we have currently works.